content/work/ui-fund.md

```diff
@@ -1,21 +1,127 @@
 ---
-title: Web Platform Features
-sub: What we did in brief @@@
+title: CSS Language Development
+sub: Cascade Layers, Container Queries, Scope, and more
 date: 2020-07-01
 logo: google
 client: &client Google
+logo: google
+image:
+  src: work/google/contain-3.jpg
+  alt: |
+    CSS Containment Module, Level 3 specification -
+    screenshot with W3C logo
+  position: top left
 list_tag: CSSWG
-people:
-  - &nicole
-    name: Nicole Sullivan
-    title: Technical Project Manager
-    venue: *client
+# people:
+#   - &nicole
+#     name: Nicole Sullivan
+#     title: Technical Project Manager
+#     venue: *client
 press:
   - text: |
-      Something nice @@@
-    <<: *nicole
+      [CSS Scope] looks amazing tbh.
+      Simple and very easy to leverage in existing models.
+      I can already see Vue SFCs using this under the hood.
+    name: Evan You
+    face:  evan-you.jpg
+    title: Creator and project lead
+    venue: Vue JS
+  - text: |
+      So happy that this work is happening.
+      Declarative style scoping will be a game changer.
+    name: Rich Harris
+    face:  rich-harris.jpg
+    title: Creator and project lead
+    venue: Svelte JS
```

Comment:
Do we want to add URLs for either of these people?

Reply:
We could. Links to the tweets, or links to the projects?

Reply:
Either way -- what have we done elsewhere? It looks to me like we're mostly linking to projects/linkedin.